### ISSUE TITLE
AttributeError ProxyFix object has no attribute 'route'

### ISSUE THREAD

**Issue by setop** [pallets/werkzeug#2520]
https://github.com/pallets/werkzeug/blob/15fcb87d36f4ed45b127692d2d739266b918503c/src/werkzeug/middleware/proxy_fix.py#L64

Using this syntax, I get `AttributeError: 'ProxyFix' object has no attribute 'route'` or `AttributeError: 'ProxyFix' object has no attribute 'register_blueprint'`.

The right syntax seems `app.wsgi_app = ProxyFix(app.wsgi_app, x_for=1, x_host=1, x_proto=1)`, which works for me.

**Comment by davidism** [comment]
These docs are talking about generic WSGI applications. For Flask, yes, use `app.wsgi_app` as the WSGI application, not `app`.

**Comment by setop** [comment]
Right, found [it for Flask](https://flask.palletsprojects.com/en/2.2.x/quickstart/?highlight=proxyfix#hooking-in-wsgi-middleware).

Easier when you know what to look for ;-)
